package.json

```diff
@@ -26,7 +26,7 @@
     "mime": "^1.6.0",
     "mkdirp": "0.5.x",
     "one-time": "0.0.4",
-    "pkgcloud": "^1.7.0",
+    "pkgcloud": "^2.1.0",
```

Comment:
Anything else in this major bump we should be worried about?

Reply:
@msluther it was @indexzero doing a big cleanup mostly with dep updates etc. It seemed safe since aws stuff wasn't touched. He removed a lot of old things that didn't exist anymore